tox.ini

```diff
@@ -15,7 +15,7 @@ isolated_build = True
 deps = -r{toxinidir}/requirements.txt
        -r{toxinidir}/requirements-test.txt
 commands =
-  pytest tests/unit tests/meta {posargs}
+  pytest tests/unit {posargs}
```

Comment:
I really don't like this change. I want the meta tests to run when the standard `py3` tests run. Now I have one more tox target to remember to run.

Also I can easily see contributors failing to run the `meta` test manually and have things fail when it hits the CI.

Reply:
@JohnVillalovos this PR isn't really fully ready for review. I will reopen it when it's done. I need to deduplicate the slow meta builds because they're now done in a virtualenv, then `meta` can be run in the standard `tox` suite. But I don't really see them as part of the unit tests, more like a custom mypy/pylint/isort job, hence the removal from the unit test target.

> Also I can easily see contributors failing to run the `meta` test manually and have things fail when it hits the CI.

I just forgot to add the meta target to the env list, I will add it when this is done.